TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/objects/properties/BaseProperties.java

```diff
@@ -13,18 +12,30 @@
     private Integer maxRam;
     private Integer keepFreeRam;
     private Double maxCpuLoad;
+    private String publicIpConfig;
 
     private BaseProperties() {
         this.maxRam = getDefaultPropertiesProvider().getMaxRam();
         this.keepFreeRam = getDefaultPropertiesProvider().getKeepFreeRam();
         this.maxCpuLoad = getDefaultPropertiesProvider().getMaxCpuLoad();
+        this.publicIpConfig = "AUTO";
```

Comment:
Please add the properety "publicIpConfig" to the ServerDefaultPropertiesProvider and use `getDefaultPropertiesProvider().getPublicIpConfig()` here